### DISCUSSION TITLE
[Project Submission] DAOInsure

### DISCUSSION THREAD

**Discussion by lopeselio** [filecoin-project/community#329] [Project Showcase]
## Project Name
DAOInsure

## Category 
Application

## Project Description

General purpose Insurance DAO for the people by the people. DAOInsure aims to solve the problem of transparency with respect to conventional Insurance reimbursements.

## Use of IPFS, Filecoin and Libp2p

- All the Data related to the claims (media- pictures) are stored on Slate, which is built on IPFS and Filecoin Network
- We use Textile to query threads and also store data to the ThreadDB built on IPFS
- The IPFS hash is then stored on chain, associated with every claim
- Decentralised chat which enables the claimer to communicate with other members of the DAO, uses Libp2p
- The project is hosted on Fleek [Fleek DAOInsure](https://daoinsure.on.fleek.co/)

Check out the project documentation and follow up with the development on our [GitHub](https://github.com/DAOInsure). All the developments and discussions related to our project will be discussed here.

**Comment by lopeselio** [comment]
We want to showcase the product in the Filecoin Newsletter.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
hey, is this going anywhere? maybe you can ping me in the slack to have a chat? @Patrick - Factor8 Solutions
